Elizabeth_Theme/src/css/_colors.scss

```diff
@@ -0,0 +1,5 @@
+$lunar-resort-orange: #dfa356;
+$lunar-resort-blue: #415fa7;
+$lunar-resort-link-teal: #00ccFF;
```

Comment:
Your variables are named `lunar-resort-x`, but your theme is not named `lunar`.  The variables should either reflect the theme's name or be generic enough to where they are not tied to the theme at all.